tests/daily_scraper/test_daily_scraper.py

```diff
@@ -43,6 +43,16 @@
     'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64-x86_64.installer.exe'
    },
 
+    # -p win32 --branch=mozilla-central --date 2013-10-01 --extension=txt
+    {'args': {'platform': 'win32',
+              'branch': 'mozilla-central',
+              'date': '2013-10-01',
+              'extension': 'txt'},
+    'target': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.txt',
+    'target_url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.txt'
+    },
+
+
```

Comment:
Whereby this is for the daily scraper and is not related to your changes. Please remove that. I happily accept additional test coverage in a separate PR, but then it should be next the Linux test for txt extension.